templates/search.html

```diff
@@ -0,0 +1,22 @@
+{% extends "_base.html" %}
+{% block main %}
+<label>
+  <p>キーワード:</p>
+  <p>{{ keyword }}</p>
+</label>
+{% for product in products_list %}
+<div>
+  <image src="{{ product.image_url }}" />
```

Comment:
```suggestion
  <image src="{{ product.image_url }}" alt="image of {{ product.title }}" />
```
と言う感じで、`alt` を追加しておきましょう

Reply:
承知しました。
修正します。